build.gradle

```diff
@@ -17,7 +17,7 @@ allprojects {
     version = project.mod_version
     archivesBaseName = project.mod_id
 
-    sourceCompatibility = targetCompatibility = JavaVersion.VERSION_17
+    sourceCompatibility = targetCompatibility = JavaVersion.VERSION_21
```

Comment:
Let's update the CI/CD pipeline to use Java 21 too please.
https://github.com/axieum/authme/blob/1ad64813853d0badce50215ec400ed65734e8450/.github/workflows/build.yml#L26-L30
https://github.com/axieum/authme/blob/1ad64813853d0badce50215ec400ed65734e8450/.github/workflows/release.yml#L47-L51